.github/workflows/publish-apple-darwin-wasm.yml

```
@@ -72,6 +72,7 @@ jobs:
       - name: Test built artifact
         if: matrix.target == 'x86_64-apple-darwin'
         run: |
+          cp ./target/${{ matrix.target }}/release/nargo ~/.cargo/bin/

```

```suggestion
          cp ./noir/target/${{ matrix.target }}/release/nargo ~/.cargo/bin/

```
These aren't run in the noir working dir, so we need to prefix it